src/main/webapp/WEB-INF/jsp/person/person_detail.jsp

```diff
@@ -39,18 +39,46 @@
         <div class="tw-space-y-12 md:tw-space-y-0 md:tw-grid tw-gap-12 tw-grid-cols-1 md:tw-grid-cols-2">
             <div class="md:tw-col-start-1 md:grid-row-start-1">
                 <uv:section-heading>
-                    <h1>
-                        <spring:message code="person.account.masterData.title"/>
-                    </h1>
+                    <jsp:attribute name="actions">
+                        <c:if test="${canEditPermissions}">
```

Comment:
canEditPermissions?! We have no permissions here

Reply:
replaced with `canEditBasedata`